asteroid/masknn/norms.py

```diff
@@ -33,16 +32,17 @@ def forward(self, x):
         Returns:
             :class:`torch.Tensor`: gLN_x `[batch, chan, *]`
         """
-        dims = list(range(1, len(x.shape)))
+        dims: List[int] = torch.arange(1, len(x.shape)).tolist()
         mean = x.mean(dim=dims, keepdim=True)
         var = torch.pow(x - mean, 2).mean(dim=dims, keepdim=True)
-        return self.apply_gain_and_bias((x - mean) / (var + EPS).sqrt())
+        value = torch.sqrt((x - mean) / (var + EPS))
```

Comment:
@mhu-coder pointed out that this is wrong, the sqrt should only be applied to the variance. He'll fix it.

Reply:
thanks for reporting it here. I have added a test and fixed the issue (in 1 commit each) but have not pushed yet.